cla_public/libs/mailgun.py

```diff
@@ -0,0 +1,22 @@
+import requests
+from flask import current_app
+
+
+def send_ref_by_email(to, subject, email_body):
+    return requests.post(
+        'https://api.mailgun.net/v3/{domain}/messages'.format(
```

Comment:
Config or local constant like [in Zendesk](https://github.com/ministryofjustice/cla_public/blob/develop/cla_public/libs/zendesk.py#L9)?